src/web/components/ArticleBody.tsx

```diff
@@ -43,42 +43,6 @@ const bodyStyle = (display: Display) => css`
         width: 100%;
         height: auto;
     }
-
-    figure {
```

Comment:
Is this completely removed because we don't have a `figure` used without styling being applied directly to the figure element?